src/osd/osd_types.cc

```diff
@@ -6700,3 +6702,61 @@ PGLSFilter::PGLSFilter() : cct(nullptr)
 PGLSFilter::~PGLSFilter()
 {
 }
+
+int PGLSParentFilter::init(bufferlist::const_iterator &params)
+{
+  try {
+    decode(parent_ino, params);
+  } catch (buffer::error &e) {
+    return -EINVAL;
+  }
+  lgeneric_subdout(cct, osd, 0) << "parent_ino=" << parent_ino << dendl;
+
```

Comment:
could take the opportunity to remove the extra empty lines.

Reply:
Dropped the extra line `PGLSPlainFilter::init()`.